contracts/insured-vesting-v1/InsuredVestingV1.sol

```diff
@@ -8,9 +8,12 @@ import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
 contract InsuredVestingV1 is Ownable {
     using SafeERC20 for IERC20;
 
+    uint256 TOKEN_RATE_PRECISION = 1e20;
```

Comment:
intentionally using 1e20 to not confuse with the expected token usage of 1e18

Reply:
Few lint warnings on line 11